<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>710</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Source Sans Pro</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Poltergeist v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/Ghost_Black.png</normaloff>../icons/Ghost_Black.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <property name="tabBarAutoHide">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="about_tab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>About</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QFrame" name="frame">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>75</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="mainTitleLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>10000</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Unispace</family>
                  <pointsize>28</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                  <underline>true</underline>
                 </font>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt;&quot;&gt;Poltergeist&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>15</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Unispace</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string>Automated keyboard and mouse inputs</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="routines_tab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>Routines</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="new_tab">
          <attribute name="title">
           <string>Create or Modify</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Source Sans Pro</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="action_auto_run"/>
    <addaction name="action_start_hidden"/>
    <addaction name="action_minimize_to_tray"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="action_info"/>
    <addaction name="action_log"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_auto_run">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Autorun at startup</string>
   </property>
  </action>
  <action name="actionShow_Notifications">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Notifications</string>
   </property>
  </action>
  <action name="actionCustom_Sounds">
   <property name="text">
    <string>Customize sounds</string>
   </property>
  </action>
  <action name="actionStart_muted">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start muted</string>
   </property>
  </action>
  <action name="action_info">
   <property name="text">
    <string>Application info</string>
   </property>
  </action>
  <action name="action_minimize_to_tray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Minimize to tray</string>
   </property>
  </action>
  <action name="action_start_hidden">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Start hidden</string>
   </property>
  </action>
  <action name="actionPTTMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Push-to-talk mode</string>
   </property>
  </action>
  <action name="actionToggleMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle mode</string>
   </property>
  </action>
  <action name="actionTogglekeybinding">
   <property name="text">
    <string>Set key for toggle mode</string>
   </property>
  </action>
  <action name="actionPTTkeybinding">
   <property name="text">
    <string>Set key for ptt mode</string>
   </property>
  </action>
  <action name="actionEnableMuteSound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>enable mute sound</string>
   </property>
  </action>
  <action name="actionEnableUnmuteSound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>enable un-mute sound</string>
   </property>
  </action>
  <action name="action_log">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
